tensorflow_model_optimization/g3doc/guide/quantization/training_example.ipynb

```diff
@@ -0,0 +1,539 @@
+{
```

Comment:
<p>MNIST model?</p><p>MNIST is a dataset. Perhaps we could say "Train a simple sequential model with the MNIST dataset using tf.keras"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/289/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done. Thanks!